src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/rc_get_capabilities_response.cc

```diff
@@ -91,6 +94,12 @@ void RCGetCapabilitiesResponse::Run() {
   }
 }
 
+void RCGetCapabilitiesResponse::onTimeOut() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  hmi_capabilities_.RemoveFromDefaultInitializedCapabilities(
```

Comment:
Response have Timeout? Are you sure?

Reply:
Also why do you RemoveFromDefaultInitializedCapabilities both in Run and OnTImeOut?

Reply:
@LuxoftAKutsan we need to know the number of received responses, and even if they are flopped by timeout, for this we delete them from `default_initialized_capabilities`, because after all responces will reseived we need set HMICooperating(true)

Reply:
Still not sure that response have a timeout

Reply:
@LuxoftAKutsan yeah. you are right this logic should be moved to the request

Reply:
@LuxoftAKutsan moved onTimeOut to requests

Reply:
BTW, did you test this implementation? I would like to have some call with brief design overview.

Reply:
@LuxoftAKutsan we don't have ATF test for this case yet, but I checked with others scripts that in case with timeout is expired, `onTimeOut` called for all requests, and now I almost finished with UTs, but I plan to supply them with separate PR with other tests for this features